pyproject.toml

```diff
@@ -100,8 +98,6 @@ LinkChecker = "^10.1.0"
 kaggle = "^1.5.12"
 tf2onnx = "^1.10.1"
 boto3 = "^1.23.5"
-paramiko = "^2.11.0"
```

Comment:
same for paramiko ?

Reply:
We don't need paramiko since some time now, not sure what it was doing here.